tools/fw_env: add redundant env support for MTD_ABSENT

Signed-off-by: Oliver Metz <oliver@freetz.org>
---
 tools/env/fw_env.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/tools/env/fw_env.c
+++ b/tools/env/fw_env.c
@@ -1164,6 +1164,9 @@ int fw_env_open(void)
 		} else if (DEVTYPE(dev_current) == MTD_UBIVOLUME &&
 			   DEVTYPE(!dev_current) == MTD_UBIVOLUME) {
 			environment.flag_scheme = FLAG_INCREMENTAL;
+		} else if (DEVTYPE(dev_current) == MTD_ABSENT &&
+			   DEVTYPE(!dev_current) == MTD_ABSENT) {
+			environment.flag_scheme = FLAG_INCREMENTAL;
 		} else {
 			fprintf (stderr, "Incompatible flash types!\n");
 			return -1;
